eZ/Publish/API/Repository/Tests/ContentServiceTest.php

```diff
@@ -536,6 +538,76 @@ public function testLoadContentInfoSetsExpectedContentInfo(ContentInfo $contentI
         );
     }
 
+    /**
```

Comment:
Not that important, but we should start making this test class less fat by splitting into smaller ones, e.g.
`\eZ\Publish\API\Repository\Tests\ContentService\ContentServiceLazyMethodsTest`